src/content/migrate/5.md

```diff
@@ -0,0 +1,156 @@
+- When using HTTP/2 and long term caching, set `optimization.splitChunks: { chunks: 'all', maxInitialRequests: 30, maxAsyncRequests: 30, maxSize: 100000 }`.
+- When using a custom configuration, replace `name` with `idHint`.
+- It was possible to turn off the defaults by setting `optimization.splitChunks: { default: false, vendors: false }`. We don't recommend doing this, but if you really want to get the same effect in webpack 5: `optimization.splitChunks: { default: false, defaultVendors: false }`.
+
+#### Cleanup the code
+
+Using `/* webpackChunkName: '...' */`: Make sure to understand the intention:
+
+- The chunk's name here is intended to be public.
+- It's not a development-only name.
+- webpack will use it to name files in production and development modes.
+- webpack 5 will automatically assign useful file names in `development` mode even when not using `webpackChunkName`.
+
+Using named exports from JSON modules: this is not supported by the new specification and you will get a warning. Instead of `import { version } from './package.json'` use `import package from './package.json'; const { version } = package;`
+
+#### Cleanup the build code
+
+- When using `const compiler = webpack(...);`, make sure to close the compiler after using it: `compiler.close();`.
+
+#### Run a single build and follow advises
+
+If there is no corresponding advise? Please create an issue and we will try to resolve it. Repeat this step until you solved at least level 3 or 4:
+
+- Level 1: Schema validation fails. Configuration options have changed. There should be a validation error with a `BREAKING CHANGE:` note.
+- Level 2: webpack exits with an error. The error message should tell you what needs to be changed.
+- Level 3: Build Errors. The error message should have a `BREAKING CHANGE:` note.
+- Level 4: Build Warnings. The warning message should tell you what can be improved.
+
+Deprecation warnings. You might get a lot of deprecation warnings. This is not a problem right now. Plugins need time to catch up with core changes. Please ignore them until release candidate out of Beta.
+
+- You can hide deprecation warnings by running node with `--no-deprecation` flag, e.g.: `node --no-deprecation node_modules/webpack/bin/webpack.js`. This should only be a temporary workaround.
+- Plugins and Loaders contributors can follow the advises in the deprecation messages to improve the code.
+
+#### Turn off ES2015 syntax in runtime code, if necessary
+
+By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `output.ecmaVersion: 5` to revert to ES5 syntax.
+
+#### Everything works?
+
+Please tweet that you have successfully migrated to webpack 5.
+
+#### It is not working?
+
+Create an issue and tell us about the issues you have encountered during the migration.
```

Comment:
It would be better to specify which repo:
```suggestion
Create an [issue](https://github.com/webpack/webpack/issues/new?template=Bug_report.md) and tell us about the issues you have encountered during the migration.
```